lib/read-file.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var fs = require('fs'),
+  q = require('q');
+
+module.exports = readFile;
+
+function readFile(filePath) {
+  return q.Promise(function (resolve, reject) {
+    fs.readFile(filePath, 'utf-8', function (error, fileContent) {
```

Comment:
Assuming that we do need to get ride of `q-io`, let's use something like `q.denodeify` or `q.nfbind` instead of wrapping it manually.

Reply:
Ooh I wasn't aware of those. That's definitely much cleaner!